app/src/main/res/values/strings.xml

```diff
@@ -1058,6 +1058,8 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_internet_access_no">No connection</string>
 
     <string name="quest_is_amenity_inside_title">Is this inside a building?</string>
+    <string name="quest_defibrillator_location">Where is this defibrillator located?</string>
+    <string name="quest_defibrillator_location_description">Please give a short but precise description of the position of this defibrillator (e.g. "in the porter's lounge")</string>
```

Comment:
"Short but precise" would be "concise", to my understanding.

Also, "of... of" does not sound so good.

```suggestion
    <string name="quest_defibrillator_location_description">Please give a concise description of this defibrillator's position (e.g. "in the porter's lounge")</string>
```

Also, you are kind of duplicating the question here. It is already clear that this is about the defi location.

Plus, it is already displayed in the hint that this is about a defi.

Reply:
Sorry, my foreign language skills are not soo good, i still didn't know 'concise' but yes, this sounds better.

I'm not sure about extending to other amenities, as they use 'location', not 'defibrillator:location'

Reply:
> I'm not sure about extending to other amenities, as they use 'location', not 'defibrillator:location'

Which ones, for example?

Reply:
Ok, after some quick research:
[location](https://wiki.openstreetmap.org/wiki/Key:location) is not used for free text input at all, only some *:location-tags.
[taginfo search](https://taginfo.openstreetmap.org/search?q=%3Alocation#keys) for :location only lists a few common :location tags
42 236 [defibrillator:location](https://taginfo.openstreetmap.org/keys/defibrillator%3Alocation)
5 402 [fire_hydrant:location](https://taginfo.openstreetmap.org/keys/fire_hydrant%3Alocation)
2 692 [generator:location](https://taginfo.openstreetmap.org/keys/generator%3Alocation)
2 397 [changing_table:location](https://taginfo.openstreetmap.org/keys/changing_table%3Alocation)
1 227 [post_box:location](https://taginfo.openstreetmap.org/keys/post_box%3Alocation)
389 [description:location](https://taginfo.openstreetmap.org/keys/description%3Alocation)

with multiple variants of defibrillator:location:[language-code]

Reply:
I see, to defibrillator is the only place where this kind of tag is used.